compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -2933,6 +2933,31 @@ public void exitBinaryEqualsExpression(BallerinaParser.BinaryEqualsExpressionCon
         this.pkgBuilder.createBinaryExpr(getCurrentPos(ctx), getWS(ctx), ctx.getChild(1).getText());
     }
 
+    @Override
+    public void exitOrderKey(BallerinaParser.OrderKeyContext ctx) {
+        if (isInErrorState) {
+            return;
+        }
+
+        boolean isAscending = true;
+        if (ctx.orderDirection() != null) {
```

Comment:
merge the two ifs

Reply:
`BLangParserListener.java` is no longer used.